### PR TITLE
Docusaurus and Architecture pattern changes

### PR DESCRIPTION
**Pull request by sguruvar**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docusaurus/observability-best-practices/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This is the source for the [AWS Observability Best Practices site](https://aws-observability.github.io/observability-best-practices/). Everyone is welcome to contribute here, not just AWS employees!
+
+## How to run/develop this site
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn [or] npm
```
Comment (bonclay7) [inline docusaurus/observability-best-practices/README.md:14]:
```suggestion
$ (yarn | npm ) install
```

File: docusaurus/observability-best-practices/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Observability Best Practices
+
+## Welcome
+
+This is the source for the [AWS Observability Best Practices site](https://aws-observability.github.io/observability-best-practices/). Everyone is welcome to contribute here, not just AWS employees!
+
+## How to run/develop this site
```
Comment (bonclay7) [inline docusaurus/observability-best-practices/README.md:7]: Maybe we should add a pre-requisite section with node, yarn or npm?

File: docusaurus/observability-best-practices/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+Using SSH:
+
+```
+$ USE_SSH=true yarn deploy [or] npm run serve
```
Comment (bonclay7) [inline docusaurus/observability-best-practices/README.md:38]: can we be break that into two commendas with one for each?

File: docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+AWS X-Ray and CloudWatch pre-date the OTEL specification, as do other leading observability solutions. However, the AWS X-Ray service readily accepts OTEL traces using ADOT. ADOT has the integrations already built into it to emit telemetry into X-Ray directly, as well as to other ISV solutions.
+
+Any transaction tracing solution requires an agent and an integration into the underlying application in order to collect signals. And this, in turn, creates [technical debt](../faq/#what-is-technical-debt) in the form of libraries that must be tested, maintained, and upgraded, as well as possibly retooling if you choose to change your solution in the future.
+
+The SDKs included with X-Ray are part of a tightly integrated instrumentation solution offered by AWS. ADOT is part of a broader industry solution in which X-Ray is only one of many tracing solutions. You can implement end-to-end tracing in X-Ray using either approach, but it’s important to understand the differences in order to determine the most useful approach for you.
+
+!!! success
```
Comment (bonclay7) [inline docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md:20]:
```suggestion
:::info
```

File: docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	We recommend instrumenting your application with the AWS Distro for OpenTelemetry if you need the following:
+
+    * The ability to send traces to multiple different tracing backends without having to re-instrument your code. For example, of you wish to shift from using the X-Ray console to [Zipkin](https://zipkin.io), then only configuration of the collector would change, leaving your applicaiton code untouched.
+
+    * Support for a large number of library instrumentations for each language, maintained by the OpenTelemetry community. 
+
```
Comment (bonclay7) [inline docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md:26]:
```suggestion
:::

```

**Review by bonclay7 — CHANGES_REQUESTED** [review]
I love this, renders really well. The only thing that doesn't are the admonitions. Can you replace `!!! success ...` and the like (tip, ...) with docusaurus admonitions `:::info ... :::` ? I tried to help but the PR is so large, it crashes my browser

https://docusaurus.io/docs/markdown-features/admonitions

File: docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md (hunk 4 above)
Comment (sguruvar) [reply docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md]: changes done

File: docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md (hunk 5 above)
Comment (sguruvar) [reply docusaurus/observability-best-practices/docs/guides/choosing-a-tracing-agent.md:27]: changes done

File: docusaurus/observability-best-practices/README.md (hunk 3 above)
Comment (sguruvar) [reply docusaurus/observability-best-practices/README.md]: changes done

File: docusaurus/observability-best-practices/README.md (hunk 2 above)
Comment (sguruvar) [reply docusaurus/observability-best-practices/README.md:7]: changes done

File: docusaurus/observability-best-practices/README.md (hunk 1 above)
Comment (sguruvar) [reply docusaurus/observability-best-practices/README.md]: changes done

**Comment by sguruvar** [comment]
all the requested changes are done

**Review by bonclay7 — APPROVED** [review]
LGTM, but we should discuss on a release plan

**Pull request merged**